packages/@react-spectrum/s2/src/ComboBox.tsx

```diff
@@ -148,6 +149,114 @@ const iconStyles = style({
   }
 });
 
+export let listbox = style<{size: 'S' | 'M' | 'L' | 'XL'}>({
+  width: 'full',
+  boxSizing: 'border-box',
+  maxHeight: '[inherit]',
+  // TODO: Might help with horizontal scrolling happening on Windows, will need to check somehow. Otherwise, revert back to overflow: auto
```

Comment:
what is the todo here? was something else being tried?

Reply:
It was something that I thought might fix the issue with the horizontal scrollbar but I'm unable to see if it actually works since I can't reproduce the original issue

Reply:
I'll make a note to test this tmrw on my windows machine

Reply:
I'm unable to reproduce the original issue as well on Chrome/FF, probably ok for now then